client-js/app/elements/labrad-registry.ts

```diff
@@ -104,7 +129,7 @@ export class LabradRegistry extends polymer.Base {
 
     this.$.pendingDialog.close()
     this.$.combinedList.selected = null;
-
+    this.kick++;
```

Comment:
This is a hack to get polymer to recompute a computed property. You'll see that `listItems` depends on kick so that when we increment here it gets recomputed. The value of kick doesn't mater, it just needs to change. We play the same trick in the node interface.